sjb/config/test_cases/test_branch_ansible_service_broker.yml

```diff
@@ -0,0 +1,137 @@
+        # this strips the -i arg to "go test". can be dropped once PR is merged:
+        # https://github.com/kubernetes-incubator/service-catalog/pull/1566 (and vendored in rebase)
+        sed -i 's/\(go test.*\)\( -i \)\(.*\)/\1 \3/' cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/test/integration.sh
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog scBuildImageTarget=.scBuildImage test-integration
+    - type: "script"
+      title: "build Service Catalog image"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        hack/build-local-images.py service-catalog
+    - type: "script"
+      title: "build Service Catalog E2E test binary"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog  bin/e2e.test
+    - type: "script"
+      title: "start cluster with Service Catalog"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        ./_output/local/bin/linux/amd64/oc cluster up --version=latest --service-catalog
+    - type: "script"
+      title: "Run Service Catalog E2E"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        sudo chmod a+rw /var/lib/origin/openshift.local.config/master/admin.kubeconfig
+        export KUBECONFIG=/var/lib/origin/openshift.local.config/master/admin.kubeconfig
+        ./_output/local/bin/linux/amd64/oc login -u system:admin
+        SERVICECATALOGCONFIG=$KUBECONFIG cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/bin/e2e.test  -v 10 -alsologtostderr 
+################################################################################
+# ASB CI ADDITIONS
+################################################################################
+    - type: "script"
+      title: "Install ASB to cluster"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        ./_output/local/bin/linux/amd64/oc create project ansible-service-broker
+        curl "https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/simple-broker-template.yaml" | ./_output/local/bin/linux/amd64/oc process -n "ansible-service-broker" -f - | ./_output/local/bin/linux/amd64/oc create -f -
+    - type: "script"
+      title: "Relist Catalog to pickup new clusterserviceclasses"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        # relistRequets is intially zero, and to trigger relist, need to increment. Patch it to force 1.
+        ./_output/local/bin/linux/amd64/oc patch clusterservicebroker ansible-service-broker -p '{"spec":{"relistRequests":1}}'
+    - type: "script"
+      title: "Configure ASB_CI_ROOT and go build environment"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        # TODO: Sanity check on configuring environment this way? Go tools? Is this the right location?
```

Comment:
It might be worth checking what GOPATH is set to before being unset on line 35 and using that.